tests/sca_image/test_output_reports.py

```diff
@@ -29,28 +31,67 @@ def test_console_output(sca_image_report):
 
     # then
     assert console_output == "\n".join(
-        ['\x1b[34msca_image scan results:', '\x1b[0m\x1b[36m', 'Passed checks: 1, Failed checks: 3, Skipped checks: 1',
```

Comment:
This is the only place where there is a test of the colors of the report.
Maybe we should add another standalone test that will detect that the colors are working correctly?